CDP4Common.NetCore.Tests/Poco/OptionTestFixture.cs

```diff
@@ -45,5 +49,31 @@ public void VerifyThatIsDefaultWorks()
             Assert.IsTrue(option1.IsDefault);
             Assert.IsFalse(option2.IsDefault);
         }
+
+        [Test]
+        public void VerifyGetNestedParameterValuesByPath()
+        {
+            var nestedElementTreeGeneratorTestFixture = new NestedElementTreeGeneratorTestFixture();
+            nestedElementTreeGeneratorTestFixture.SetUp();
+
+            var option = nestedElementTreeGeneratorTestFixture.iteration.Option.Single(x => x.ShortName == "OPT_A");
+
+            var doubleParameters = option.GetNestedParameterValuesByPath<double>("Sat\\m\\\\OPT_A", nestedElementTreeGeneratorTestFixture.domainOfExpertise).ToList();
```

Comment:
i propose we use the `@` syntax for these strings, that makes them literal strings that don't require escaping. Since the tests are not only about testing but also self-documenting the code, it can serve as intended use.